src/Flame/Types.purs

```diff
@@ -20,7 +22,8 @@ type VNodeData = {
         -- we need attrs mainly for svg
         attrs :: Object String,
         props :: Object String,
-        on :: VNodeEvents
+        on :: VNodeEvents,
+        hook :: Object Foreign
```

Comment:
Should we call it `hooks` since we have `attrs` and `props`?

Reply:
VNodeData is the snabbdom's node structure, so the naming is intentional. It won't work otherwise.